packages/flutter_tools/lib/src/build_system/targets/ios.dart

```diff
@@ -670,6 +791,7 @@ class DebugIosApplicationBundle extends IosAssetBundle {
     const CheckDevDependenciesIos(),
     const DebugUniversalFramework(),
     const DebugIosLLDBInit(),
+    const _IssueLaunchRootViewControllerAccess(),
```

Comment:
This will only run in Debug mode. If we want it to run in all build modes (profile, release), add it as a dependency of `IosAssetBundle` instead